### PR TITLE
3 get api categories

### PR DESCRIPTION
**Pull request by LKFULLER1**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+const app = express();
+app.use(express.json());
+
+app.get('/api/categories', getCategories);
+
+
+app.all('/*', (req, res) => {
+  res.status(404).send({ msg: 'Route not found' });
+});
```
Comment (dav3rid) [inline app.js:14]: This is great - do you have a test for this?

**Review by dav3rid — CHANGES_REQUESTED** [review]
Well done for deleting the commented out code - just get a test in for your 404

File: app.js (hunk 1 above)
Comment (LKFULLER1) [reply app.js:14]: nope but I can add one!

Comment (LKFULLER1) [reply app.js:14]: added a test for the 404 error

**Pull request merged**